### PR TITLE
add option to use BC platform IDs

### PR DESCRIPTION
**Pull request by mikeurbanski1**
# In This PR

- Adds an option to show BC_ IDs in the scan results (and also in the skip comments)

## Pictures/videos

- [X] I've reviewed my own code

### CODE REVIEW

**Review by kartikp10 — APPROVED** [review]
Nifty! Looks good to me.

File: src/configuration.ts [hunk 1]
```diff
@@ -20,3 +20,9 @@ export const getPathToCert = (): string | undefined => {
     const pathToCert = configuration.get<string>('certificate');
     return pathToCert;
 };
+
+export const getUseBcIds = (): boolean | undefined => {
+    const configuration: vscode.WorkspaceConfiguration = vscode.workspace.getConfiguration('checkov');
+    const useBcIds = configuration.get<boolean>('useBridgecrewIDs');
```
Comment (SteveVaknin) [inline src/configuration.ts:26]: Perhaps we should have a default fallback?

**Review by SteveVaknin — APPROVED** [review]
Looks good, left one comment 👌

File: src/configuration.ts (hunk 1 above)
Comment (mikeurbanski1) [reply src/configuration.ts]: Good call

**Pull request merged**